src/course-unit/course-xblock/CourseXBlock.test.jsx

```diff
@@ -309,4 +310,34 @@ describe('<CourseXBlock />', () => {
       expect(getByText(visibilityMessage)).toBeInTheDocument();
     });
   });
+
+  it('displays a render error message if item has error', async () => {
```

Comment:
[nit]: It looks like this test shouldn't be asynchronous

Reply:
Changed, thanks